wallet/how-to/use-non-evm-networks/starknet-snap/connect-to-starknet.md

```diff
@@ -1,6 +1,74 @@
 ---
-description: Connect
+description: Connect your Starknet account to MetaMask
 sidebar_position: 1
 ---
 
-# Connect
\ No newline at end of file
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Connect your Starknet account to MetaMask
+
+To connect a dapp to MetaMask for StarkNet, you can use `get-starknet`. 
+This tool simplifies the process of integrating Starknet with MetaMask and allows you to connect your wallets and install the necessary Snap.
+After set up, the Snap enables secure and efficient interaction with the Starknet ecosystem.
```

Comment:
We should add the `wallet_invokeSnap` instructions to this page instead of the Starknet landing page. So include a brief blurb here in the intro explaining the use case for each method.